app/helpers/date_time_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module DateTimeHelper
+  def format_time(datetime)
```

Comment:
Perhaps this could be renamed to `format_time_in_twelve_hour_clock`

Reply:
I've opted for `format_time_12_hour_clock`.